src/attribute.rs

```diff
@@ -0,0 +1,13 @@
+use super::*;
+
+#[derive(EnumString)]
+#[strum(serialize_all = "kebab_case")]
+pub(crate) enum Attribute {
+  NoExitMessage,
+}
+
+impl Attribute {
+  pub(crate) fn from_lexeme(lexeme: &str) -> Option<Attribute> {
+    lexeme.parse().ok()
+  }
```

Comment:
I think this is a little clearer, since attributes should only be parsed from naems:

```suggestion
  pub(crate) fn from_name(lexeme: Name) -> Option<Attribute> {
    name.lexeme().parse().ok()
  }
```